erpnext_documentation/www/docs/user/manual/en/manufacturing/subcontracting.md

```diff
@@ -61,6 +61,15 @@ Click on Create > Purchase Receipt from the Purchase Order. Set the Accepted and
 
 <img class="screenshot" alt="Sub-Contracting" src="{{docs_base_url}}/assets/img/manufacturing/subcontract5.png">
 
+### 1.5 Supplier Sourced Raw Material
+If some raw material is sourced by the Supplier directly, such raw material has to be marked in the BOM.
+
+* It will have zero value in BOM
```

Comment:
how will the cost of this item be factored into total manufacturing costs?
if supplier is providing the item, the manufacturer is still paying the costs for it right?

Reply:
@prasadarr It is for the sub-contract and from the user perspective. This kind of raw material is not in the user's stock but it is part of the BOM because it is being used in the manufacture. Suppliers can choose to put it in their Sales Order or it can be included in the price of the manufactured good. But for the user, it has zero value in BOM.